solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,76 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
+We can create a graph based off the roads between the farms which is given as input.
```

Comment:
could you put this in the CPP section (the java and py solution does not use dfs)

Reply:
Would it be preferable to have 2 separate solutions? (I'd rather have language differences be just transcription, not an entirely different soln)